README.md

```diff
@@ -159,6 +159,8 @@ Benchmark
 ------------
 If configured with `--enable-benchmark` (which is the default), binaries for benchmarking the libsecp256k1 functions will be present in the root directory after the build.
 
+For an accurate benchmark, it is strongly recommended to pin the process to a dedicated CPU core and to disable CPU frequency scaling.
```

Comment:
This will be more reliable at best, but not more "accurate", since it doesn't reflect actual usage.